File: src/classes/Parceiro.java
package classes;

import java.util.ArrayList;

public class Parceiro {
	private int id;
	private String nome;
	private String cpf;
	private String cnpj;
	private boolean tipo;//false para Pessoa Física, true para Pessoa Jurídica
	private String email;
	private String usuario;
	private ArrayList<Telefone> telefones;
	private ArrayList<Endereco> enderecos;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	
	public String getCpf() {
		return cpf;
	}
	public void setCpf(String cpf) {
		this.cpf = cpf;
	}
	
	public String getCnpj() {
		return cnpj;
	}
	public void setCnpj(String cnpj) {
		this.cnpj = cnpj;
	}
	
	public boolean isTipo() {
		return tipo;
	}
	public void setTipo(boolean tipo) {
		this.tipo = tipo;
	}
	
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	
	public String getUsuario() {
		return usuario;
	}
	public void setUsuario(String usuario) {
		this.usuario = usuario;
	}
	
	public ArrayList<Telefone> getTelefones() {
		return telefones;
	}
	public void setTelefones(ArrayList<Telefone> telefones) {
		this.telefones = telefones;
	}
	
	public ArrayList<Endereco> getEnderecos() {
		return enderecos;
	}
	public void setEnderecos(ArrayList<Endereco> enderecos) {
		this.enderecos = enderecos;
	}
}

File: API C#/ListaPublica/ListaPublica/Classes/Parceiro.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ListaPublica.Classes
{
    public class Parceiro
    {
        public Parceiro()
        {
            telefones = new List<Telefone>();
            enderecos = new List<Endereco>();
        }

        public int id { get; set; }
        public string nome { get; set; }
        public string cpf { get; set; }
        public string cnpj { get; set; }
        public bool tipo { get; set; }
        public string email { get; set; }
        public string usuario { get; set; }
        public IList<Telefone> telefones { get; set; }
        public IList<Endereco> enderecos { get; set; }

    }
}
